mscore/instrdialog.cpp

```diff
@@ -88,9 +88,8 @@ void InstrumentsDialog::on_saveButton_clicked()
             info.setFile(info.filePath() + ext);
       QFile f(info.filePath());
       if (!f.open(QIODevice::WriteOnly)) {
-            QString s = tr("Open Instruments File\n%1\nfailed: ")
-               + QString(strerror(errno));
-            QMessageBox::critical(mscore, tr("Open Instruments File"), s.arg(f.fileName()));
+            QString s = tr("Open Instruments File\n%1\nfailed: %2").arg(f.fileName()).arg(strerror(errno));
```

Comment:
No. Touching the entire code base (just like this PR does in fact...) it's not good... it just mess up the entire history. To me the history is a lot more valuable than a white space too much or a QObject too much...